src/OpenTelemetry.Sampler.AWS/FallbackSampler.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="FallbackSampler.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Sampler.AWS;
+
+internal class FallbackSampler : Trace.Sampler
```

Comment:
This implementation of the FallbackSampler is just here for placeholder. It will eventually be a rate limited sampler with 1 req/sec and 5% of remaining requests. 
I will update it in the next and final PR.